_data/changelogs/component-text-input.yml

```
@@ -5,6 +5,21 @@ type: component
 changelogURL:
 
 items:
+  - date: NNNN-NN-NN
+    summary: Textarea supports validation.
```

This makes me think about all types of validation, rather than the validation component. I might be misunderstanding though! Does something like this feel right?

```suggestion
    summary: Added `textarea` support to the validation component. 
```